app/src/main/res/drawable/ic_share_white_24dp.xml

```diff
@@ -4,6 +4,6 @@
     android:viewportHeight="24.0"
     android:viewportWidth="24.0">
     <path
-        android:fillColor="@color/md_white_1000"
+        android:fillColor="@color/md_grey_800"
```

Comment:
Unneeded change

Reply:
Icon wasn't visible in white therefor I made change.

Reply:
@iamareebjamal Sir, what to do about this, shall I remove it or consider it ?
When you long press on chat message then share icon will be visible.

Reply:
Remove it